src/main/resources/assets/neoforge/lang/en_us.json

```
@@ -194,6 +194,7 @@
   "neoforge.configgui.advertiseDedicatedServerToLan.tooltip": "Set this to true to enable advertising the dedicated server to local LAN clients so that it shows up in the Multiplayer screen automatically.",
   "neoforge.configgui.attributeAdvancedTooltipDebugInfo": "Additional Attribute Advanced Tooltips",
   "neoforge.configgui.attributeAdvancedTooltipDebugInfo.tooltip": "Set this to true to enable additional information about attributes on an item when advanced tooltips is on.",
+  "neoforge.configgui.enableB3DValidationLayer": "When enabled, all usage of B3D will be validated against allowed usage",
```

This should also have a tooltip version for the config GUI. I would also avoid using the B3D abbreviation as this might be unfamiliar to random users.

For example:

```suggestion
  "neoforge.configgui.enableB3DValidation": "Enable Blaze3D Validation Layer",
  "neoforge.configgui.enableB3DValidation.tooltip": "When enabled, all usage of Blaze3D (Minecraft's internal rendering abstraction) will be validated against allowed usage.",
```